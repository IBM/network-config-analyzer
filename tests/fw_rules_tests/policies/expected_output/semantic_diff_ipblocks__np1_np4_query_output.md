|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|semantic_diff, config1: np1, config2: np4, key: Added connections between persistent peers||||||
||[default,kube-system-dummy-to-ignore,vendor-system]|[*]|[kube-system]|[tier=frontend]|All connections|
||[kube-system]|[!has(tier) or tier=not_frontend_for_demo]|[kube-system]|[tier=frontend]|All connections|
|semantic_diff, config1: np1, config2: np4, key: Removed connections between persistent peers||||||
||[kube-system]|[tier=frontend]|[default,kube-system-dummy-to-ignore,vendor-system]|[*]|All connections|
||[kube-system]|[tier=frontend]|[kube-system]|[!has(tier) or tier=not_frontend_for_demo]|All connections|
|semantic_diff, config1: np1, config2: np4, key: Added connections between persistent peers and ipBlocks||||||
|||0.0.0.0-9.255.255.255|[kube-system]|[tier=frontend]|All but UDP 53|
|||11.0.0.0-172.20.255.255|[kube-system]|[tier=frontend]|All but UDP 53|
|||172.22.0.0-172.29.255.255|[kube-system]|[tier=frontend]|All but UDP 53|
|||172.31.0.0-255.255.255.255|[kube-system]|[tier=frontend]|All but UDP 53|
|||10.0.0.0/8|[kube-system]|[tier=frontend]|All connections|
|||172.21.0.0/16|[kube-system]|[tier=frontend]|All connections|
|||172.30.0.0/16|[kube-system]|[tier=frontend]|All connections|
|semantic_diff, config1: np1, config2: np4, key: Removed connections between persistent peers and ipBlocks||||||
||[kube-system]|[tier=frontend]||0.0.0.0-49.49.255.255|All connections|
||[kube-system]|[tier=frontend]||49.50.0.1/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.11/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.13/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.15/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.17-255.255.255.255|All connections|
||[kube-system]|[tier=frontend]||49.50.0.3/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.5/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.7/32|All connections|
||[kube-system]|[tier=frontend]||49.50.0.9/32|All connections|
