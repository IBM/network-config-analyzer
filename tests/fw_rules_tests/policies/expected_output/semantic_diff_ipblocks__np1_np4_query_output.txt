np1 and np4 are not semantically equivalent.

Added connections between persistent peers (based on topology from config: np4) :
src_ns: [default,kube-system-dummy-to-ignore,vendor-system] src_pods: [*] dst_ns: [kube-system] dst_pods: [tier=frontend] conn: All connections
src_ns: [kube-system] src_pods: [!has(tier) or tier=not_frontend_for_demo] dst_ns: [kube-system] dst_pods: [tier=frontend] conn: All connections

Removed connections between persistent peers (based on topology from config: np1) :
src_ns: [kube-system] src_pods: [tier=frontend] dst_ns: [default,kube-system-dummy-to-ignore,vendor-system] dst_pods: [*] conn: All connections
src_ns: [kube-system] src_pods: [tier=frontend] dst_ns: [kube-system] dst_pods: [!has(tier) or tier=not_frontend_for_demo] conn: All connections

Added connections between persistent peers and ipBlocks (based on topology from config: np4) :
src: 0.0.0.0/0 dst_ns: [kube-system] dst_pods: [tier=frontend] conn: All but UDP 53
src: 10.0.0.0/8,172.21.0.0/16,172.30.0.0/16 dst_ns: [kube-system] dst_pods: [tier=frontend] conn: All connections

Removed connections between persistent peers and ipBlocks (based on topology from config: np1) :
src_ns: [kube-system] src_pods: [tier=frontend] dst: 0.0.0.0-49.49.255.255,49.50.0.1,49.50.0.3,49.50.0.5,49.50.0.7,49.50.0.9,49.50.0.11,49.50.0.13,49.50.0.15,49.50.0.17-255.255.255.255 conn: All connections
